### PR TITLE
Result page

### PR DESCRIPTION
**Pull request by ChhikaraBRUH**
Added Result Page

Preview:  https://deploy-preview-3--snkrs-quiz-screens.netlify.app/pages/result.html

### CODE REVIEW

**Review by Ayushi20-19 — APPROVED** [review]
good use of sematic html... also neat code

File: pages/result.html [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+		<section class="question-section">
+			<div class="question-container">
+				<div class="question">Q1. Which famous rapper collaborated with Nike, and then Adidas, to create Yeezy?</div>
+
+				<div class="options-container">
+					<div class="question-option">Jay Z</div>
+					<div class="question-option correct-option">Kanye West</div>
+					<div class="question-option">Drake</div>
+					<div class="question-option">Eminem</div>
```
Comment (shaik-saad) [inline pages/result.html:50]: You can use inputs for the answer options. Also you can name these classes as answer-option, makes is understandable why that class is used.

**Review by shaik-saad — APPROVED** [review]
Added a suggestion, do check it out. Also your UI looks clean and good.

File: pages/result.html (hunk 1 above)
Comment (ChhikaraBRUH) [reply pages/result.html]: Changed the class names as requested. I will have to see how to use inputs and refactor the code later accordingly.

**Pull request merged**